components/forms/EmailForm.tsx

```diff
@@ -48,6 +59,123 @@ export const EmailForm = (props: EmailFormProps) => {
         onChange={handleEmailChange}
         onFocus={handleEmailChange}
       />
+      {isSuccessOpen && (
+  <ModalConfirmation
+    isOpen={isSuccessOpen}
+    onClose={toggleSuccess}
+    body={
+      <div
+        style={{
```

Comment:
Can we remove the inline styling and move this to tailwind classes.
For a rule of thumb - unless its something that is exceptionally hard to do with Tailwind, don't use inline styles 🙏